chapters/methodology/methodology.tex

```diff
@@ -5,8 +5,10 @@ \chapter{Methodology}
 deep-penetration radiation transport problems. In general, the variance reduction
 parameters generated by CADIS and FW-CADIS are not sufficient for
 problems that are strongly
-anisotropic with respect to the flux. The method will extend existing
-methods to generate importance maps--that in turn generate variance reduction
+anisotropic with respect to the flux. The new method will extend existing
```

Comment:
I can't figure out what this response means in relation to the shown code snippet. I was just cutting out words that seemed like too many. If you think they need to be there, than keep them.